model-providers/ollama/runtime/src/main/java/io/quarkiverse/langchain4j/ollama/OllamaDefaultToolsHandler.java

```diff
@@ -0,0 +1,132 @@
+package io.quarkiverse.langchain4j.ollama;
+
+import static io.quarkiverse.langchain4j.ollama.ChatRequest.Builder;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+import dev.langchain4j.agent.tool.ToolExecutionRequest;
+import dev.langchain4j.agent.tool.ToolParameters;
+import dev.langchain4j.agent.tool.ToolSpecification;
+import dev.langchain4j.data.message.AiMessage;
+import dev.langchain4j.internal.Json;
+import dev.langchain4j.model.input.Prompt;
+import dev.langchain4j.model.input.PromptTemplate;
+
+public class OllamaDefaultToolsHandler implements ToolsHandler {
+
+    static final PromptTemplate DEFAULT_SYSTEM_TEMPLATE = PromptTemplate.from("""
+            You have access to the following tools:
+
+            {tools}
+
+            You must always select one of the above tools and respond with a JSON object matching the following schema,
+            and only this json object:
+            {{
```

Comment:
Why the double `{`?

Reply:
copy past from lanchain python version, I've try with one and it works